doc/Changelog.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
-# Release 1.9 (07-07-2015) {#Release19}
+## git master {#master}
+
+* Use program name as default window title
+* Implemented eq::x11::Window and eqCPU example as a template for
+  CPU-based rendering, for example for raytracing
```

Comment:
was already part of previous release: https://github.com/Eyescale/Equalizer/pull/437